RestaurantReservation.Db/Models/Employee.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace RestaurantReservation.Db.Models;
+public class Employee
+{
+    [Key]
+    public int EmployeeId { get; set; }
+    [Required]
+    public int RestaurantId { get; set; }
+    [Required, MaxLength(50)]
+    public string FirstName { get; set; }
+    [Required, MaxLength(50)]
+    public string LastName { get; set; }
+    [Required, MaxLength(50)]
+    public string Position { get; set; }
+
+    public Restaurant Restaurant { get; set; }
```

Comment:
NIIT: when dealing with strings, objects, lists, .. it is better to specify the property as nullable if it is, or otherwise put the required keyword or you can also manually initialize

    public required Restaurant Restaurant { get; set; }

or

public Restaurant Restaurant { get; set; } = new();